src/main/resources/index.html

```diff
@@ -1,3 +1,3 @@
 <div>
```

Comment:
I think this could be deleted now that you have added `index.jelly`.

Reply:
=> #12

Reply:
Thanks!